src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -1047,6 +1047,11 @@ public static bool ShouldInclude(this ITypeDefinitionMember member, EmitContext
                 }
             }
 
+            if (method != null && (context.Module.PEEntryPoint == method || context.Module.DebugEntryPoint == method))
```

Comment:
>|| context.Module.DebugEntryPoint == method [](http://example.com/codeflow?start=73&length=43)

Do we have a test affected by this DebugEntryPoint logic?

Reply:
Added tests for that, thanks.